wallets/provider-trezor/src/helpers.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+import { Networks } from '@rango-dev/wallets-shared';
+import TrezorConnect from '@trezor/connect-web';
+
+const ETHEREUM_CHAIN_ID = '0x1';
```

Comment:
Considering that this line is duplicate with a line in ledger-provider, I think you can move it to "@rango-dev/wallets-shared" package.